### PR TITLE
Fix:Respect 'in: query' and 'in: requestBody' from the microservice definition

### PR DESCRIPTION
**Pull request by kego1992**

### CODE REVIEW

File: asyncy/processing/Services.py [hunk 1]
```diff
@@ -207,8 +207,17 @@ def get_command_conf(cls, story, chain):
         hostname = await Containers.get_hostname(story, line, chain[0].name)
         args = command_conf.get('arguments')
         body = {}
+        query_params = {}
+        path_params = {}
+
         for arg in args:
-            body[arg] = story.argument_by_name(line, arg)
+            value =story.argument_by_name(line, arg)
```
Comment (judepereira) [inline asyncy/processing/Services.py]: Space - please run the pep8 tool by installing `tox`, and then run `tox` in the directory of the engine.

File: asyncy/processing/Services.py [hunk 2]
```diff
@@ -207,8 +207,17 @@ def get_command_conf(cls, story, chain):
...
+            if args[arg]['in'] == 'query':
+                query_params[arg] = value
+            elif args[arg]['in'] == 'path':
+                path_params[arg] = value
+            else:
```
Comment (judepereira) [inline asyncy/processing/Services.py]: Explicitly check for `requestBody`, in all other cases, log a warning.

File: asyncy/processing/Services.py [hunk 3]
```diff
@@ -217,9 +226,9 @@ def get_command_conf(cls, story, chain):
                 'Content-Type': 'application/json; charset=utf-8'
             }
         }
-
+        
         port = command_conf['http'].get('port', 5000)
-        path = command_conf['http']['path']
+        path = HttpUtils.add_params_to_url(command_conf['http']['path'].format(**path_params), query_params)
```
Comment (judepereira) [inline asyncy/processing/Services.py]: Lint.

**Comment by kego1992** [comment]
Sure Jude will do the required. Will check about the CI failures

File: asyncy/processing/Services.py (hunk 1 above)
Comment (kego1992) [reply asyncy/processing/Services.py]: Thanks, tox command helped. I have done the suggested changes

**Comment by kego1992** [comment]
Cool , will get it done

File: tests/unit/processing/Services.py [hunk 4]
```diff
@@ -184,14 +184,28 @@ def test_resolve_chain(story):
...
+            },
+            'default_case': {
+                'type': 'string',
+                'in': 'invalid'
+            }
         }
     }
 
-    expected_url = 'http://container_host:2771/invoke'
+    expected_url = 'http://container_host:2771/invoke/bar?query_param=bar'
```
[the remark this answers is not in the document]
Comment (kego1992) [reply tests/unit/processing/Services.py]: I have added the required changes, let me know if the test case was updated as expected

[the remark this answers is not in the document]
Comment (judepereira) [reply tests/unit/processing/Services.py]: @kego1992 Fantastic! It looks good. I'll review it fully tonight, and merge it if looks all good :)

**Pull request merged**